packages/core/http/core-http-router-server-internal/src/router.ts

```diff
@@ -237,7 +238,7 @@ export class Router<Context extends RequestHandlerContextBase = RequestHandlerCo
     if (excludeVersionedRoutes) {
       return this.routes.filter((route) => !route.isVersioned);
     }
-    return [...this.routes];
+    return this.routes;
```

Comment:
probably unnecessary but this might remove a potential mutation guard we accedentaly had

Reply:
Ah man, I somehow missed this. That's a good point, will fix.